cfme/fixtures/appliance.py

```diff
@@ -199,9 +200,10 @@ def get_vddk_url(provider):
     except (KeyError, AttributeError):
         pytest.skip('VDDK URL/Version not found in cfme_data.basic_info')
     if url is None:
-        pytest.skip("There is no vddk url for this VMware provider version")
-    else:
-        return url
+        logger.warning("Using VDDK {}, as VDDK {} was unavailable".format("v6_5", vddk_version))
+        url = conf.cfme_data.basic_info.vddk_url.get("v6_5")
```

Comment:
Its not guaranteed this key will be available either. This is a good fallback lookup, but the skip clause still needs to exist.

Reply:
```
try:
    vddk_urls =  conf.cfme_data.basic_info.vddk_url
except (KeyError, AttributeError):
    pytest.skip('no vddk urls available')
key = vddk_version
if vddk_version not in vddk_urls:
    logger.warning('using default vddk url')
    key = 'v6_5'
url = vddk_urls.get(key)
```

I'm thinking a structure like this is what we want here, so that we can log and set defaults, and skip on vddk's not available.